OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -22,12 +22,14 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 			var sequences = MiniYaml.MergeLiberal(map.SequenceDefinitions, Game.ModData.Manifest.Sequences.Select(s => MiniYaml.FromFile(s)).Aggregate(MiniYaml.MergeLiberal));
 
 			foreach (var actorInfo in map.Rules.Actors)
-				foreach (var renderInfo in actorInfo.Value.Traits.WithInterface<RenderSimpleInfo>())
+			{
+				foreach (var renderInfo in actorInfo.Value.Traits.WithInterface<RenderSpritesInfo>())
 				{
 					var image = renderInfo.Image ?? actorInfo.Value.Name;
```

Comment:
The check will cover some more rendering traits that are lower level such as crates.